### DISCUSSION TITLE
Optimise internal AutoCue for Pop / Rock radio stations

### DISCUSSION THREAD

**Discussion by wegGehamstert** [AzuraCast/AzuraCast#7610] [General]
Hey, my name is Niklas from the webradio BreathFM

I'm using Azuracast since a few years now and i love the idea of AutoCue. 
I thougth it may be interesting to see how other stations manage to get seamless transitions because often those fail on my end. 
If you wanna hear my attempt you can tune in at: https://stream.breathfm.de/listen/mainstream/high.mp3

**Comment by gAlleb** [comment]
Hi, I've listened for a few songs and transitions were quite nice. But stream is very loud (at -9 LUFS).

<img width="353" alt="Снимок экрана 2024-12-08 в 14 50 59" src="https://github.com/user-attachments/assets/7efec9e0-1c9a-4abe-8d80-88029ca49b32">

Internal autocue should take care of your transitions. And probably it does fine.

**Comment by wegGehamstert** [comment]
Thanks for tuneing in, i think this is due Stereotool, maybe i need to change this a little bit.
